Packs/AncientGrain/Playbooks/RH_-_ChatGPT_Ancient_Grain_Recipes.yml

```diff
@@ -0,0 +1,333 @@
+        labelarg:
+          simple: Please Specify Ancient Grain
+        options: []
+        optionsarg:
+        - simple: amaranth
+        - simple: quinoa
+        - simple: teff
+        - simple: farro
+        - simple: spelt
+        placeholder: ""
+        readonly: false
+        required: false
+        tooltip: ""
+        type: singleSelect
+      sender: ""
+      title: Specify Ancient Grain
+      totalanswers: 0
+    id: "3"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    message:
+      bcc: null
+      body:
+        simple: Is the ancient grain already collected?
+      cc: null
+      format: ""
+      methods: []
+      subject: null
+      timings:
+        completeafterreplies: 1
+        completeaftersla: false
+        completeafterv2: true
+        retriescount: 2
+        retriesinterval: 360
+      to: null
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: a968ecb2-e2d3-4dd7-8346-cb23f69e0894
+      iscommand: false
+      name: Collect Ancient Grain
+      type: collection
+      version: -1
+    taskid: a968ecb2-e2d3-4dd7-8346-cb23f69e0894
+    timertriggers: []
+    type: collection
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 545
+        }
+      }
+  "4":
+    continueonerrortype: ""
+    id: "4"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "5"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      marcancientgrain:
+        simple: ${Specify Ancient Grain.Answers.0}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: commands.local.cmd.set.incident
+      id: 0e37c512-d678-4f44-858a-738c494dbafa
+      iscommand: true
+      name: Set Ancient Grain
+      script: Builtin|||setIncident
+      type: regular
+      version: -1
+    taskid: 0e37c512-d678-4f44-858a-738c494dbafa
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 720
+        }
+      }
+  "5":
+    continueonerrortype: ""
+    id: "5"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "7"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      extend-context:
+        simple: =.
+      prompt:
+        simple: Provide a recipe using the ancient grain of ${incident.marcancientgrain}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Perplexity LLM Custom
+      description: Send Text Message as a prompt to Perplexity
+      id: 38757eb4-8751-4b0b-8bd5-2d3c76396543
+      iscommand: true
+      name: Send Ancient Grain to ChatGPT for Recipe
+      script: Perplexity LLM Custom|||perplexity-send-prompt
```

Comment:
didn't find this command in our repository - can you please share more details on this?